.github/workflows/ci.yaml

```diff
@@ -79,6 +79,7 @@ jobs:
         env:
           # Set the OFE_SLOW_TESTS to True if running a Cron job
           OFE_SLOW_TESTS: ${{ fromJSON('{"false":"false","true":"true"}')[github.event_name != 'pull_request'] }}
+          DUECREDIT_ENABLE: 'yes'
```

Comment:
@IAlibay what's the value in having this as an option on the matrix?  It's optional as-is

Reply:
@richardjgowers I think I messed up on the fixture

So by default duecredit is disabled, so if you run the test it'll give you an error because openfe.due.due.active == False.

You need to have this env variable enabled in order for duecredit to kick in.

Reply:
right, but why do we need the option to toggle duecredit tests?  Having it behind an `importorskip` is fine, this is just fluff in the CI matrix that weighs it down.

Reply:
this is a duecredit env variable, not our own invention